Dangerfile

```diff
@@ -24,6 +24,8 @@ end
 
 ## Let's check if there are any changes in the project folder
 has_app_changes = !git.modified_files.grep(/Sources/).empty?
```

Comment:
Just change this location to `/Sources/StreamChat/` and it'll work, no need to add additional flag

When we're ready (for StreamChatUI) we'll change it back to this

To avoid ruby error, put ' (make it `'/Sources/StreamChat/'`)

Reply:
Great, thank you, will fix this in next commit (uploaded wrong again)